sdk/spring/build-developing-version-artifacts-if-needed.md

```diff
@@ -0,0 +1,28 @@
+
+### Released version
+
+We **highly recommend** you refer to the `README.md` files of the released version.
+
+Here are the steps to get `README.md` files of the released version:
+
+1. On GitHub, navigate to the main page of the repository.
+1. Click the `master` button, then click `Tags`.
+1. In the `Find a tag` text box, input and enter the starter module name you want to search, such as `azure-spring-boot-starter-xxx`.
+1. Click one of the searched tags. (You will find that the address in your browser bar changes to that specific version. You can manually edit the address with other versions, then quickly switch to other versions.)
+1. Then check the README.md file there.
```

Comment:
Hi, @michaelqi793 , I updated the contents. Could you please add screenshots for these steps?